docs/en/upload.md

```diff
@@ -62,11 +62,12 @@ print("Connecting to WiFi access point...")
 wifi.setmode(wifi.STATION)
 wifi.sta.config(SSID, PASSWORD)
 -- wifi.sta.connect() not necessary because config() uses auto-connect=true by default
-tmr.alarm(1, 1000, 1, function()
+wifitimer = tmr.create()
+wifitimer:alarm(1000, tmr.ALARM_AUTO, function(cb_timer)
     if wifi.sta.getip() == nil then
         print("Waiting for IP address...")
     else
-        tmr.stop(1)
+        cb_timer:unregister()
```

Comment:
I didn't test this, but it looks like a reference to `wifitimer` will linger on...

Perhaps the example should have a `wifitimer = nil` after this line to fully dispose of all the references so that it can be GC'ed. 

But this code would be better by avoiding the assignment to the `wifitimer` variable in the first place.